tests/hilti/codegen/type-info.hlt

```diff
@@ -173,22 +173,23 @@ public function tuple<strong_ref<TestOptionals>, TypeInfo> makeTestOptionals() {
 
 using namespace hilti::rt;
 
-namespace __hlt::Test {
+namespace __hlt_::Test {
```

Comment:
Now that I see this, I think it's worth tracking down where this changes comes from and fix it. The `__hlt` is actually configurable, and there shouldn't be anything added to it.

Reply:
I added a commit addressing this.